index.js

```diff
@@ -16,9 +16,19 @@ function promisifyOwnProperties(obj) {
 }
 
 module.exports = obj => {
+  if (!obj) {
+    return obj;
+  }
+
+  const typeOfObj = typeof obj;
+
+  if (typeOfObj !== 'function' && (typeOfObj !== 'object' || Array.isArray(obj))) {
```

Comment:
Hmm, passing an array in makes a weird object.

I kept the test with an array just to see what happened. Is this wanted?

```
    Expected value to be (using ===):
      [1, true, "string", null, undefined, Symbol(some symbol)]
    Received:
      {"0": 1, "1": true, "2": "string", "3": null, "4": undefined, "5": Symbol(some symbol)}
```